daffodil-runtime1/src/main/scala/org/apache/daffodil/externalvars/ExternalVariablesLoader.scala

```diff
@@ -66,13 +68,15 @@ object ExternalVariablesLoader {
 
   def fileToBindings(file: File): Queue[Binding] = {
     Assert.usage(file ne null)
-    ExternalVariablesValidator.validate(file) match {
+    val validatorResult = ExternalVariablesValidator.validate(file)
+    validatorResult match {
       case Left(ex) => Assert.abort(ex)
       case Right(_) => // Success
     }
     val enc = determineEncoding(file) // The encoding is needed for ConstructingParser
     val input = scala.io.Source.fromURI(file.toURI)(enc)
-    val node = ConstructingParser.fromSource(input, true).document.docElem
```

Comment:
Add comment that this was changed from the Constructing loader because of a unexpected error, and reusing the DaffodlXMLLoader works, no need for a special case loader for external variable bindings.